python/src/spark_rapids_ml/classification.py

```diff
@@ -512,46 +543,48 @@ def _transformEvaluate(
         return scores
 
 
-from typing import Callable
-
-from pyspark.ml.param.shared import HasFeaturesCol, HasLabelCol, HasPredictionCol
-from pyspark.sql.types import (
-    ArrayType,
-    DoubleType,
-    FloatType,
-    IntegerType,
-    StringType,
-    StructField,
-    StructType,
-)
-
-from .core import (
-    FitInputType,
-    _CumlEstimatorSupervised,
-    _CumlModelWithPredictionCol,
-    param_alias,
-)
-from .params import HasFeaturesCols, _CumlClass, _CumlParams
-from .utils import PartitionDescriptor, _ArrayOrder, _concat_and_free
-
-
 class LogisticRegressionClass(_CumlClass):
     @classmethod
     def _param_mapping(cls) -> Dict[str, Optional[str]]:
-        return {}
+        return {
+            "maxIter": "max_iter",
+            "regParam": "C",  # regParam = 1/C
+            "tol": "tol",
+            "fitIntercept": "fit_intercept",
+            "elasticNetParam": "",
+            "threshold": "",
+            "thresholds": "",
+            "probabilityCol": "",
+            "rawPredictionCol": "",
+            "standardization": "",
+            "weightCol": "",
+            "aggregationDepth": "",
+            "family": "",
+            "lowerBoundsOnCoefficients": "",
+            "upperBoundsOnCoefficients": "",
+            "lowerBoundsOnIntercepts": "",
+            "upperBoundsOnIntercepts": "",
+            "maxBlockSizeInMB": "",
```

Comment:
Some of these should probably be set to `None` (i.e. raise an error if supplied), e.g.  `standardization`, `weightCol`, etc (not sure of full list).

Reply:
Revised. Thanks!